README.md

```diff
@@ -391,3 +391,15 @@ To have NervesHub invoke it, update your `config.exs` as follows:
 ```elixir
 config :nerves_hub, client: MyApp.NervesHubClient
 ```
+
+### Enabling remote IEx access
+
+It's possible to remotely log into your device via the NervesHub web interface. This
+feature is disabled by default. To enable, adding the following to your `config.exs`:
```

Comment:
Dang. I was too late - This should probably be `add the following..` instead of `adding`